pkg/transformers/json/utils.go

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) Mainflux
```

Comment:
This is purely for time manipulation, so `time.go` is more suitable name than `utils.go`.